### ISSUE TITLE
Feature Request: Add dropIgnoreErrors to DDLInvoker

### ISSUE THREAD

**Issue by hvesalai** [slick/slick#93]
When developing a system I like to drop and recreate tables every now and then. It is also usefull in unit tests.

Because the schema changes when developing, some of the database objects might not exist leading to errors when dropping.

I'm using this and it might be worthwhile having it in the core slick:

```
def dropIgnoreErrors(implicit session: Backend#Session): Unit =
  for(s <- ddl.dropStatements) {
    try {
      session.withTransaction {
        session.withPreparedStatement(s)(_.execute)
      }
    } catch { 
      case e => 
    }
  }
```

**Comment by sap1ens** [comment]
Still an issue in 3.0.0-RC1 :( 

DatabaseAction fails **silently** with `org.postgresql.util.PSQLException: ERROR: table "******" does not exist`

**Comment by AlexITC** [comment]
is there any progress?

**Comment by davegurnell** [comment]
Related point. It would be great to give developers access to `DROP IF EXISTS` in databases that support it.

**Comment by cvogt** [comment]
@davegurnell looks like the above PR is very close to delivering this

**Comment by hvesalai** [comment]
Merged #1704 to master
